modules/osdk-cli-ref-run-bundle-upgrade.adoc

```diff
@@ -23,7 +23,14 @@ The `run bundle-upgrade` subcommand upgrades an Operator that was previously ins
 |`-n`, `--namespace` (string)
 |If present, namespace in which to run the CLI request.
 
+|`--security-context-config <security_context>`
+|Specifies the security context to use for the catalog pod. Allowed values include `restricted` and `legacy`. The default value is `legacy`. ^[1]^
+
 |`-h`, `--help`
 |Help output for the `run bundle` subcommand.
 
 |===
+[.small]
+--
+1. The `restricted` security context is not compatible with the `default` namespace. To configure your Operator's pod security admission in your production environment, see "Complying with pod security admission". For more information about pod security admission, see "Understanding and managing pod security admission".
```

Comment:
```suggestion
1. The `restricted` security context is not compatible with the `default` namespace. To configure the pod security admission of your Operator in your production environment, see "Complying with pod security admission." For more information about pod security admission, see "Understanding and managing pod security admission."
```
I don't know how far we need to go to avoid possessives, but for you consideration. Also, unless we have a rule saying otherwise, periods go inside quotes from a grammar perspective.

Reply:
Thanks for the suggestions, @ShaunaDiaz! The [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives) recently lightened it's rules around possessives. Previously, we were told to avoid possessives, but now they are permitted if it makes the sentence more direct and easier to read.

I agree with you about the quotation marks, but unfortunately the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks#quotation-marks-with-other-punctuation) dictates the punctuation should be outside of the quotation marks.